include/arch/riscv/arch/32/mode/machine.h

```diff
@@ -9,9 +9,20 @@
 #include <util.h>
 #include <arch/model/smp.h>
 #include <stdint.h>
+#include <plat/machine/devices_gen.h>
+
+/*
+ * Currently all RISC-V 32-bit platforms supported have the mtime register
+ * mapped at the same offset of the base address of the CLINT.
+ */
+#define CLINT_MTIME_OFFSET 0xbff8
```

Comment:
Could you move this into `#ifdef CONFIG_RISCV_USE_CLINT_MTIME` also?

Reply:
Resolved